### ISSUE TITLE
disabled attribute seems to reset slides

### ISSUE THREAD

**Issue by kmturley** [zefoy/ngx-swiper-wrapper#241]
If you change the disabled attribute to `true` then the slides completely reset:
`[disabled]="swiperDisabled" `

It means the current slide is not selected anymore.

I found a workaround by accessing Swiper directly using:
```
import { SwiperDirective } from 'ngx-swiper-wrapper';

export class Component {
  swiperDirective: SwiperDirective;

  @ViewChild(SwiperDirective)
  set swiper(directive: SwiperDirective) {
    this.swiperDirective = directive;
  }

  disableSwiper(disabled) {
    const swiper = this.swiperDirective.swiper();
    swiper.allowSlideNext = !disabled;
    swiper.allowSlidePrev = !disabled;
  }
```

This retains the currently select slide

**Comment by sconix** [comment]
Yes the disabled property is for disabling Swiper fully, i.e. unloading it. So if you just want to disable some functionality of the Swiper you should do it as you have or via using Swiper configuration options.

**Comment by kmturley** [comment]
Ah ok, is it worth documenting how to access the Swiper instance for custom functionality?

**Comment by sconix** [comment]
The method for accessing is mentioned in the README and example app has an example how to use it. I know the documentation could be better, but I have decided to use all the time I have to spear for upkeeping the code instead of documentation.
